Dockerfile

```diff
@@ -73,8 +67,10 @@ RUN apt update \
         /var/tmp \
         ~/.cache \
         ; do [ ! -d $dir ] || find $dir -mindepth 1 -delete; done
-ENV PATH=$PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
-COPY --from=bnfc-build /var/tmp/bnfc/bnfc /usr/local/bin/
+RUN cabal update \
+    && cabal install alex happy \
+    && cabal install BNFC
```

Comment:
I'm little concerned here if jflex and BNFC will be the right version. What version will be BNFC?

jflex depends on ubuntu distro and BNFC on GHC version. I managed to find stable solution with nix but I'm not sure if it's easy to use it as part of this Docker image.
https://github.com/rchain/rchain/tree/dev/nix#bnfc

Reply:
https://github.com/rchain/buildenv/pull/6/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557L6
The latest bnfc version installed by cabal already include this bnfc commit. The old dev tried to build it from scratch because the old bnfc doesn't include this commit.

Reply:
I tried this process many times actually.

Reply:
Anyway, I removed that commit if you feel uncomfortable.
I only need latest git version

Reply:
I'm fine with the change if BNFC version is correct.